app/components/BudgetGrid/index.js

```diff
@@ -22,25 +29,60 @@ export class BudgetGrid extends React.Component<BudgetGridProps> {
   render() {
     const { transactions, categories } = this.props;
 
-    return (
-      <table className={styles.budgetGrid}>
-        <thead>
-          <tr>
-            <th>Category</th>
-            <th>Description</th>
-            <th>Amount</th>
-          </tr>
-        </thead>
-        <tbody>
-          {transactions.map((transaction: Transaction): React.Element<any> => (
-            <BudgetGridRow key={transaction.id} transaction={transaction} categories={categories} />
-          ))}
-        </tbody>
+    const ResponsiveCell = ({ columnName, value, className }: ResponsiveCellProps) => (
+      <div className={`${styles.responsiveCell} ${className || ''}`}>
+        <div className={styles.cellLabel}>{columnName}</div>
+        <div className={styles.cellContent}>{value}</div>
+      </div>
+    );
+
+    const TableComponent = ({ children, ...otherProps }) => (
+      <table {...otherProps}>
```

Comment:
is this intentionally `table` and not `Table`?

Reply:
Yes, that' intentionally `table`. 
`TableComponent` here is overriding the default `TableComponent` used by `Table`.
We do this to render a custom table footer.